tests/tests/fv3_esg_HAFS_v0_hwrf_thompson

```diff
@@ -30,7 +30,7 @@ export FHMAX=12
 DT_ATMOS="300"
 
 export FV3_RUN=esg_HAFS_v0_hwrf_run.IN
-export CCPP_SUITE=HAFS_v0_hwrf_thompson
+export CCPP_SUITE=FV3_HAFS_v0_hwrf_thompson
```

Comment:
Do we still need these global atm only tests?

Reply:
I recommend keeping this test. It is for an alternate physics suite in HAFS.